Source/SpacePort/XUnitTest.Test/RestSharpTest.cs

```diff
@@ -0,0 +1,78 @@
+﻿using RestSharp;
+using SpacePort;
+using System;
+using System.Collections.Generic;
+using Xunit;
+using Newtonsoft.Json;
+using System.Linq;
+
+namespace XUnitTest.Test
+{
+    public class RestSharpTest
+    {
+        [Fact]
+        public void TestSimpleGet()
+        {
+            var client = new RestClient("https://swapi.co/api/");
+            var request = new RestRequest("people/", Method.GET);
+
+            var content = client.Execute(request).Content;           
+        }
```

Comment:
Does this test something? Can you test without an Assert? Or what is the expectation? :O